ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -351,6 +351,28 @@ tests:
       YQ_VERSION: 4.30.5
     workflow: openshift-e2e-ibmcloud-heterogeneous-upi
   timeout: 6h0m0s
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
+  cron: 20 23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes and
+        not @fips'
+      GO_VERSION: 1.20.7
+      JQ_VERSION: "1.6"
+      LOKI_ENABLED: "false"
+      OCP_VERSION: "4.14"
+      RESOURCE_GROUP: ipi-cicd-resource-group
+      SIZE_VARIANT: large
+      TERRAFORM_VERSION: 1.5.5
+      WORKERS: "2"
+      YQ_VERSION: 4.30.5
+    workflow: openshift-e2e-ibmcloud-heterogeneous-cucushift
+  timeout: 8h0m0s
```

Comment:
Hey Trevor/Jeremy: Is it possible in increase to 10 hrs? Our install is 1hr30 + CucuShift is 8hrs.  Thoughts? Paul

Reply:
did you try deleting this line at all? We have jobs with timeouts set to 23h in openshift-tests-private.